### PR TITLE
docs: add Java docs and change code style to Google Code Style

### PR DESCRIPTION
**Pull request by arvgord**
(#13)

- Changed code style
- Added Java docs in changed classes

### CODE REVIEW

**Comment by arvgord** [comment]
This is the last code style PR)

File: src/main/java/com/quantumsoft/qupathcloud/gui/windows/NewDatasetWindow.java [hunk 1]
```diff
@@ -34,87 +35,102 @@
 import javafx.stage.Stage;
 import javafx.stage.StageStyle;
 
-import java.util.List;
-
-
+/**
+ * The NewDatasetWindow will be presented if user click on the New Dataset button.
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/gui/windows/NewDatasetWindow.java]: clicks on the?

File: src/main/java/com/quantumsoft/qupathcloud/gui/windows/NewDicomStoreWindow.java [hunk 2]
```diff
@@ -22,81 +22,99 @@
...
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
 import javafx.scene.paint.Color;
 import javafx.stage.Modality;
 import javafx.stage.Screen;
 import javafx.stage.Stage;
 import javafx.stage.StageStyle;
 
+/**
+ * The NewDicomStoreWindow will be presented if user click on the New DICOM Store button.
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/gui/windows/NewDicomStoreWindow.java]: clicks on the?

File: src/main/java/com/quantumsoft/qupathcloud/gui/windows/SynchronizationWindow.java [hunk 3]
```diff
@@ -20,75 +20,92 @@
...
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.RowConstraints;
 import javafx.stage.Modality;
 import javafx.stage.Screen;
 import javafx.stage.Stage;
 
+/**
+ * The SynchronizationWindow will be presented if user click on the Synchronize button.
+ */
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/gui/windows/SynchronizationWindow.java:34]: clicks?

**Review by poojavenkatram — COMMENTED** [review]
Looks great thanks Artyom :) Couple of super small comments...

**Comment by arvgord** [comment]
Thanks! I've updated current PR.

**Review by poojavenkatram — APPROVED** [review]
Thanks :) LGTM.

**Pull request merged**